content/docs/getting-started/installation/helm/modules/replication/install-repctl.md

```diff
@@ -45,14 +45,24 @@ You can start using Container Storage Modules for Replication with help from `re
       ./repctl cluster add -f "/root/.kube/config-1","/root/.kube/config-2" -n "cluster-1","cluster-2"
       ```
    > **_NOTE:_**  If using a single Kubernetes cluster in a stretched configuration there will be only one cluster.
-3. Install replication controller and CRDs:
+3. Install replication CRDs:
       ```shell
       ./repctl create -f ../deploy/replicationcrds.all.yaml
+      ```
+4. Install replication controller:
+
+   Update `allow-pvc-creation-on-target` arg to `true` or `false` as required.  
+   Default: `false`
+
+     `true`: It will replicate the PVC on target cluster (in case of `multi cluster`)
```

Comment:
Change "will replicate" to "replicates".

Reply:
done